package.json

```diff
@@ -28,7 +28,7 @@
   "author": "Brigade Engineering",
   "license": "MIT",
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0"
```

Comment:
Yeah maybe this was too aggressive. What do you think @lencioni? You're right @jmeas that there's nothing right now that will break. One issue though is that we're not really testing with react 14. So if things go south, we'll likely find out about it from an issue. And at that time, it's going to be a little hard to say "oh, you know what, we'll just cut support for React 14". :man_shrugging: